cypress/support/common_commands.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * Sets the page data range
```

Comment:
date range?

Reply:
Thank you for the catch, it has been corrected!